components/shex-cmumps-to-rdf.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * File:  shex-cmumps-to-rdf.js
+ */
+
+// var _ = require('underscore');
+
+// var rs = require("rdfstore");
+var N3 = require("n3");
+// var Promise = require("bluebird");
+var jsonld = require("jsonld");
+
+var fs = require('fs');
+var util = require('util');
+
+var shexiface = require("../lib/shex/shexiface");
+var wrapper = require('../src/javascript-wrapper');
+
+module.exports = wrapper( updater );
+
+function updater(data) {
+    // shexiface.now("enter updater");
+    parsedData = JSON.parse(data);
```

Comment:
I think this line would be better off as var parsedData = _.isString(data) ? JSON.parse(data) : data;